### PR TITLE
[RFC]scripts: add a build scripts for public signed fw

### PR DESCRIPTION
**Pull request by xiulipan**
Add a simple scripts for sof-bin release binary

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+# checkout branch
+git checkout -b "stable-$RELEASE_VERSION"
+
+# define folders
+SOF_FW_DIR="lib/firmware/intel/sof/$RELEASE_VERSION"
+PUBLIC_SIGNED_DIR="$SOF_FW_DIR/public-signed"
+INTEL_SIGNED_DIR="$SOF_FW_DIR/intel-signed"
+TPLG_DIR="lib/firmware/intel/sof-tplg-$RELEASE_VERSION"
+TOOLS_DIR="tools/$RELEASE_VERSION"
+
```
Comment (lgirdwood) [inline build.sh:75]: @dbaluta you guys will need to update this after it's merged for your FW and topology layouts.

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+       -v release version
+       -m path to MEU tool
+       -a Build all platforms
+EOF
+}
+
+while getopts "s:v:m:a" OPTION; do
+  case "$OPTION" in
+    s) SOF_DIR=$OPTARG ;;
+    v) RELEASE_VERSION=$OPTARG ;;
```
Comment (lgirdwood) [inline build.sh:40]: Should this be release git tag instead of version ?

**Review by lgirdwood — COMMENTED** [review]
@xiulipan can you put the usage in the comments. i.e. I guess this has to run from master branch on sof-bin ? I would also expect this script to checkout the release tag in sof repo too.

File: build.sh (hunk 2 above)
Comment (xiulipan) [reply build.sh:40]: @lgirdwood Let me check if the sof and sof-bin share the same tag or version things.

**Comment by xiulipan** [comment]
close as this will not cover the intel-signed fw copy

**Pull request closed without merging**